### ISSUE TITLE
Gitea build is failing

### ISSUE THREAD

**Issue by AbdulrhmnGhanem** [kitspace/gitea#13]
Running `docker-compose build gitea` resulted in build failure for both `Dockerfile` and `Dockerfile.gitea.dev`.

```
ERROR in ./web_src/js/features/stopwatch.js 11:0-43
Module not found: Error: Can't resolve 'pretty-ms' in '/go/src/code.gitea.io/gitea/web_src/js/features'
 @ ./web_src/js/index.js 73:0-56 3242:207-220

webpack 5.11.0 compiled with 1 error in 184773 ms
make: *** [Makefile:690: public/js/index.js] Error 1
ERROR: Service 'gitea' failed to build: The command '/bin/sh -c if [ -n "${GITEA_VERSION}" ]; then git checkout "${GITEA_VERSION}"; fi  && make clean-all build' returned a non-zero code: 2
```

**Comment by kasbah** [comment]
Fixed with 7849d6c6db21e22b678f59e7c30b52c559ddb6a6?

**Comment by AbdulrhmnGhanem** [comment]
Yes
